src/index.ts

```diff
@@ -11,7 +12,30 @@ export {AudioPlayer as Plugin};
 export {VERSION, NAME};
 
 const pluginName = 'audioPlayer';
+registerPlugin(pluginName, AudioPlayer);
 
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-registerPlugin(pluginName, AudioPlayer);
+
+const AudioPreset = {
+  template: (): any => miniAudioUI({player: {}, config: {}}),
+  condition: (): boolean => true
+};
+
+// Custom Preset has to be added in the bundle phase (before kalturaPlayer instance and as a result the UI is created and the )
+if (!window.__kalturaplayerdata) {
```

Comment:
you can also do

window.__kalturaplayerdata = window.__kalturaplayerdata || {};
window.__kalturaplayerdata.ui = window.__kalturaplayerdata.ui || {};
window.__kalturaplayerdata.ui.customPreset = window.__kalturaplayerdata.ui.customPreset || [];
window.__kalturaplayerdata.ui.customPreset.push(AudioPreset);